client/components/App.jsx

```diff
@@ -69,23 +73,24 @@ class App extends Component {
             <ProductDetail
               product={currentProduct}
               updateProduct={(id) => this.updateProduct(id)}
-              productReviews={this.state.reviewData}
+              productReviews={reviewData}
+              addToCart={(obj) => this.addToCart(obj)}
             />
-            {this.state.relatedProducts &&
+            {this.state.relatedProducts && (
```

Comment:
might make sense to continue your destructure paradigm on line 62 for consistency